README.md

```diff
@@ -29,3 +29,117 @@ The installer scripts also work fine without docker; just run the following two
 
 The unpacking requires recent versions of msitools (0.98) and libgcab
 (1.2); sufficiently new versions are available in e.g. Ubuntu 19.04.
+
+
+# Build instructions for native environments
```

Comment:
I'd still prefer to use a different term than "native environments" as it's a quite unclear term in this case.

"Build instructions for local installation" maybe?

Reply:
Agreed. I didn't realize "native" here can mean Windows or Linux non-Docker.

It can cause confusion. I went with your suggestion